src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/Domain/ICanStoreCommitmentInformation.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.ProviderPayments.Calc.PaymentsDue.Domain
```

Comment:
what's the difference between `ICanStoreCommitmentInformation` and `IHoldCommitmentInformation`.

Reply:
One has getters, one has setters